plugins/vbe/meson.build

```diff
@@ -0,0 +1,36 @@
+if get_option('plugin_vbe').enabled() or \
+   (get_option('plugin_vbe').auto() and libfdt.found())
+cargs = ['-DG_LOG_DOMAIN="Vbe"']
+
+shared_module('fu_plugin_vbe',
+  fu_hash,
+  sources : [
+    'vbe-simple.c',
+    'fu-plugin-vbe.c',
+    'fit.c',
+    'fit_test.c'
```

Comment:
I think this should be fu-vbe-test.c and actually run as a meson test -- and also using the GTest framework.

Reply:
OK well we need to decide on this. If this fit stuff is going to live in fwupd and not use the external library for some years, it seems best to rewrite it.

But can I at least avoid adding glib stuff to it? That would change the API (header file) which I would very much like to keep the same between the different users.

Reply:
OK I figured it out. Done